lib/components/user/user-account-screen.js

```diff
@@ -87,7 +87,10 @@ class UserAccountScreen extends Component {
     // On exit, route to default search route.
     this.props.routeTo('/')
   }
-
+  /**
+   * Save changes and return to the planner.
+   * @param {*} userData The user edited state to be saved, provided by Formik.
+   */
   _handleExitAndSave = async userData => {
```

Comment:
Minor tweak: could you change this name to `_handleSaveAndExit` (just to match its behavior)?

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/239/commits/6902c2b4c80245cb0821fdf0b69ec5406dbb242f